components/cascader/index.tsx

```diff
@@ -3,6 +3,7 @@ import RcCascader from 'rc-cascader';
 import arrayTreeFilter from 'array-tree-filter';
 import classNames from 'classnames';
 import omit from 'omit.js';
+import isEqual from 'lodash/isEqual';
```

Comment:
Try `shallowequal`

Reply:
I can't find `shallowequal` in package.json, need I set up it?

Reply:
lodash isEqual is ok then.